src/components/Time/AutoRefresh.vue

```diff
@@ -0,0 +1,100 @@
+<template></template>
+
+<script>
+import axios from "axios";
+import SaxonJS from "saxon-js";
+export default {
+  data() {
+    return {
+      interval: setInterval(this.fetchData, 90000),
+      layers: [],
+      xsltTime: `parse-xml($xml)//Layer[not(.//Layer) and Name = 'REPLACE_WITH_LAYERNAME']!map
+                      {
+                          'Dimension' : map
+                          {
+                              'Dimension_time' : string(Dimension[@name = 'time']),
+                              'Dimension_time_default' : string(Dimension[@name = 'time']/@default),
+                              'Dimension_ref_time' : string(Dimension[@name = 'reference_time'])
+                          }
+                      }`,
+    };
+  },
+  mounted() {
+    this.$root.$on("timeLayerAdded", (layerName) => {
+      this.layers.push(layerName);
+    });
+    this.$root.$on("timeLayerRemoved", (layer) => {
+      this.layers = this.layers.filter((l) => l !== layer.get("layerName"));
+    });
+  },
+  beforeDestroy() {
+    this.stopPolling();
+  },
+  methods: {
+    checkRefresh(layersInfo) {
+      let errorList = [];
+      layersInfo.forEach((layerInfo) => {
+        if (
+          layerInfo["layer"].get("layerDimensionTime") !==
+            layerInfo["layerData"].Dimension.Dimension_time ||
+          layerInfo["layer"].get("layerDimensionRefTime") !==
+            layerInfo["layerData"].Dimension.Dimension_ref_time
+        ) {
+          errorList.push(layerInfo["layer"]);
+        }
+      });
+      if (errorList.length !== 0) {
+        this.$root.$emit("refreshExpired", errorList);
+      }
+    },
+    stopPolling() {
+      clearInterval(this.interval);
+    },
+    async fetchData() {
```

Comment:
Rename to `fetchLayerData()`, `fetchLayersInfo()` or `getCapabilitiesOfLayers()` or something more specific to match purpose of function?